.gitignore

```diff
@@ -12,6 +12,8 @@ configuration.yaml
 *.iws
 .idea/
 bin/
+lib/
+out/
```

Comment:
Can you create a ticket in jira so that we can add these items to the web-api-skeleton during the next sprint?